src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -1,7 +1,8 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
-Description: |
-  TODO: TIG-3324
+Description: >
+  This workload sequentially issues the same query in various combinations of projection and index
+  presence.
```

Comment:
```suggestion
  This workload measures the performance of queries with rich filters. First, we issue a bunch of plain queries with rich filter to get the performance baseline. After that, we issue the same query, but with projection. Finally, we create an index and issue the same query with projection, expecting it will use this index.
```